TACC/protx-dashboard
# Task/COOKS-295: improve deployment

## Overview: ##

Improve deployment by populating volumes with some services.

The same change will be done in a PR to Core-Portal-Deployments repo.

## Related Jira tickets: ##

* [COOKS-295](https://jira.tacc.utexas.edu/browse/COOKS-295)

## Testing Steps: ##
1. Edit docker-compose.yml so that you'll be running the frontend that is distributed in the image (4 changes are needed.)
2. docker-compose up
3. ```
    docker exec core_portal_django python3 manage.py migrate
    docker exec core_portal_django python3 manage.py collectstatic --noinput
    docker exec core_portal_cms python3 manage.py migrate
    docker exec core_portal_cms python3 manage.py collectstatic --noinput
4. `docker exec -it protx python3 scripts/run_queries.py`
5. Go to `https://cep.dev/protx/dash/maltreatment` and make sure things work
